### PR TITLE
Fix the broken CI by disabling warmup by default

### PR DESCRIPTION
**Pull request by xuzhao9**
We should not run warmup rounds to get eager latency when eager mode is already used.
However, in non-eager mode, we need to do the following:
1. Run correctness rounds with fp32 precision to get accurate eager output
2. Run warmup rounds with default precision to get accurate eager latency
3. Run dynamo rounds to get the correctness result

Note that the warmup rounds here are different from the warmup iterations in `run.py`. Here, we use a few warmup rounds to approximate eager latency, which is later used to calculate PT2 compilation time. In `run.py`, we run warmup rounds for performance testing.

Fix https://github.com/pytorch/benchmark/issues/1581

### CODE REVIEW

File: torchbenchmark/util/model.py
Comment (janeyx99) [inline torchbenchmark/util/model.py:154]: How does should_check_correctness become different for eager vs pt2? This seems to suggest that we do not check correctness for pt2?

**Review by janeyx99 — APPROVED** [review]
Stamping to unblock, but we should see green CI before landing

File: torchbenchmark/util/model.py
Comment (xuzhao9) [reply torchbenchmark/util/model.py:154]: We do not check correctness for eager, because we use eager as the baseline to calculate the correctness of pt2.

Comment (janeyx99) [reply torchbenchmark/util/model.py:154]: Oh sorry yes I got that flipped. Would it be good to include a comment about that somewhere?

Comment (xuzhao9) [reply torchbenchmark/util/model.py:154]: Yep I can update when we have the CI result

**Pull request closed without merging**
